### PR TITLE
2886 wallet erc20 api

### PR DESCRIPTION
**Pull request by cyberliem**

### CODE REVIEW

File: accounting/wallet-erc20/http/http.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	"go.uber.org/zap"
+
+	"github.com/KyberNetwork/reserve-stats/accounting/wallet-erc20/storage"
+	"github.com/KyberNetwork/reserve-stats/lib/httputil"
+	_ "github.com/KyberNetwork/reserve-stats/lib/httputil/validators" // import custom validator functions
+	ethereum "github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	maxTimeFrame     = time.Hour * 24 * 365 * 1 // 1 year
```
Comment (halink0803) [inline accounting/wallet-erc20/http/http.go]: Does not need to * 1

Comment (cyberliem) [reply accounting/wallet-erc20/http/http.go]: fixed

Comment (halink0803) [reply accounting/wallet-erc20/http/http.go]: @cyberliem I think you are fixing not all the places.

Comment (favadi) [reply accounting/wallet-erc20/http/http.go]: that redundant * 1 is fine by me.

File: accounting/wallet-erc20/http/http.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	sugar *zap.SugaredLogger
+	r     *gin.Engine
+	host  string
+	st    storage.Interface
+}
+
+type getTransactionsQuery struct {
+	httputil.TimeRangeQuery
+	Wallet string `form:"wallet" binding:"required,isAddress"`
+	Token  string `form:"token" binding:"required,isAddress"`
```
Comment (favadi) [inline accounting/wallet-erc20/http/http.go]: wallet/token should not be required, they should act like filter.

**Pull request merged**